docs/sql-api.md

```diff
@@ -504,9 +504,12 @@ function boolean **_prom_catalog.create_exemplar_table_if_not_exists**(metric_na
 ### _prom_catalog.create_ingest_temp_table
 Creates a temporary table (if it doesn't exist) used for ingestion of metrics or traces.
 Temporary table is created using supplied table and schema as prototype.
-Suppresses corresponding DDL logging, otherwise PG log may get unnecessarily verbose.Returns temporary table name
+Suppresses corresponding DDL logging, otherwise PG log may get unnecessarily verbose.
+Api user has to make sure that table_prefix is unique per session/connection.
```

Comment:
As I commented [here](https://github.com/timescale/promscale/pull/1462#discussion_r916731637), the prefix must be globally unique.

Reply:
Not sure that's true. Temporary tables are bound to session/connection. It's by PG design.

Reply:
Basically temporary tables from other sessions are invisible so no need for global uniqueness.

Reply:
Ya, you're right. Sorry I missed that.